homeassistant/config_entries.py

```diff
@@ -104,7 +104,11 @@ def __new__(cls: type[_T], value: str, recoverable: bool) -> _T:
 
     @property
     def recoverable(self) -> bool:
-        """Get if the state is recoverable."""
+        """Get if the state is recoverable.
+
+        If the entry is state is recoverable, unloads
```

Comment:
If the entry state is recoverable,

Reply:
I will adjust, and should probably write a blog post for this as well.   I will do both after some sleep as it is getting late here.

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/1365